presto-main/src/test/java/com/facebook/presto/sql/query/TestSubqueries.java

```diff
@@ -89,13 +89,13 @@ public void testCorrelatedExistsSubqueriesWithOrPredicateAndNull()
     public void testUnsupportedSubqueriesWithCoercions()
```

Comment:
rename this test `testSubqueriesWithCoercions` since this is supported now

Reply:
fixed